src/Microsoft.DncEng.SecretManager/SecretTypes/GitHubAppSecret.cs

```diff
@@ -93,9 +100,10 @@ public override async Task<List<SecretData>> RotateValues(Parameters parameters,
             DateTimeOffset rollOn = _clock.UtcNow.AddMonths(6);
 
             return new List<SecretData> {
-                    new SecretData(appId, DateTimeOffset.MaxValue, DateTimeOffset.MaxValue),
+                    new SecretData(appId, DateTimeOffset.MaxValue, rollOn),
```

Comment:
Shouldn't it be max value as it never changes?

Reply:
That is technically correct, but but giving these values different dates doesn't really do anything, they all get rotated when the earliest one says to.